metsim/methods/mtclim.py

```diff
@@ -26,74 +21,61 @@ def run(forcing: pd.DataFrame, disagg=True):
     lat_idx = forcing.index.names.index('lat')
     time_idx = forcing.index.names.index('time')
     sg = solar_geom(forcing['elev'][0], forcing.index.levels[lat_idx][0])
+    sg = {'tiny_rad_fract' : sg[0],
```

Comment:
I'll add this.  Just a note, I have it set up so that numba doesn't allow object mode, and will fail at compile time.